mail-client/evolution [-spell] : cmake still requires gtkspell3

> -- Checking for module 'gtkspell3-3.0'
--   No package 'gtkspell3-3.0' found
CMake Error at cmake/modules/PkgConfigEx.cmake:32 (message):
  Necessary libraries not found or not enough version.  If you want to
  disable gtkspell usage, please use -DENABLE_GTKSPELL=OFF argument to cmake
  command.

Seems like missing switch in ebuilds `src_configure`.

--- a/mail-client/evolution/evolution-3.28.0.ebuild
+++ b/mail-client/evolution/evolution-3.28.0.ebuild
@@ -102,6 +102,7 @@ src_configure() {
 	local mycmakeargs=(
 		-DENABLE_YTNEF=OFF
 		-DENABLE_PST_IMPORT=OFF
+		-DENABLE_GTKSPELL=$(usex spell ON OFF)
 	)
 	cmake-utils_src_configure
 }

